src/symbolize/mod.rs

```diff
@@ -117,6 +118,29 @@ pub use symbolizer::Symbolizer;
 use crate::Addr;
 
 
+/// A enumeration of the different input types the symbolization APIs
+/// support.
+#[derive(Clone, Copy, Debug)]
+pub enum Input<A, O> {
+    /// A virtual address.
+    ///
+    /// A virtual address is an address as a process would see it, for
+    /// example. A virtual address may contain relocation or address space
+    /// randomization effects.
+    VirtAddr(A),
+    /// A file address.
+    ///
+    /// A file address is an address as it would appear in a binary or
```

Comment:
it's not just about relocations, so it feels a bit too much to go into specific (but incomplete) details. Maybe we can just say that this is address (or offset if you agree with me below) that matches ELF file (and DWARF) symbol and segment information.